asmcli/lib/parse.sh

```diff
@@ -245,6 +245,10 @@ parse_args() {
         context_set-option "PRINT_VERSION" 1
         shift 1
         ;;
+      --disable_ui)
+        context_set-option "UI_DISABLED" 1
```

Comment:
I think for all of the other options the environment variable exactly matches the option name, so I think "DISABLE_UI" is more consistent.

Reply:
Done.